SHREK Job Submission 2022-07-18 20:31:17.625153
cmd args: 
Namespace(yaml=['tests/pythia8-charm-simulation-full/runCalorimeter.yaml', 'tests/pythia8-charm-simulation-full/runCharmSimu.yaml', 'tests/pythia8-charm-simulation-full/runMinbSimu.yaml', 'tests/pythia8-charm-simulation-full/runPass3Track.yaml', 'tests/pythia8-charm-simulation-full/runPass4Track.yaml', 'tests/pythia8-charm-simulation-full/runPileup.yaml'], tag='sP22a-hfcharm', offset=0, submit=True, check=True, handshake=True, vo='wlcg', site='BNL_OSG_SPHENIX', prodSourceLabel='test', workingGroup='sphenix', user='jwebb2', branch='test')
subdir: 
/direct/sphenix+u/jwebb2/work/2022/SHREK/sPHENIX-test-production/sP22a-hfcharm
tag: 
sP22a-hfcharm-1896cb4c-06fa-11ed-9977-00163e101049
check:
b'INFO : uploading workflow sandbox\nINFO : check workflow user.jwebb2.sP22a-hfcharm-1896cb4c-06fa-11ed-9977-00163e101049\nINFO : messages from the server\n2022-07-19 00:31:16.917703 : start /var/cache/pandaserver/workflow.11dbee30-0b48-4a8d-a70c-f680744ba82e\n2022-07-19 00:31:17.409402 : downloading sandbox from https://pandaserver-doma.cern.ch:25443/cache/jobO.18af8d70-6190-448d-80a4-e79bef96c797.tar.gz\n2022-07-19 00:31:17.451896 : parse workflow\n2022-07-19 00:31:17.488605 : failed to run with while constructing a mapping\n  in "/tmp/tmphqm69nyp/sP22a-hfcharm-workflow.cwl", line 13, column 5\nfound duplicate key "outputSource" with value "Pythia8Pass3Track/outDS" (original value: "Pythia8Calorimeter/outDS")\n  in "/tmp/tmphqm69nyp/sP22a-hfcharm-workflow.cwl", line 15, column 5\n\nTo suppress this check see:\n    http://yaml.readthedocs.io/en/latest/api.html#duplicate-keys\n\nDuplicate keys will become an error in future releases, and are errors\nby default when using the new API.\n Traceback (most recent call last):\n  File "/opt/panda/lib64/python3.6/site-packages/pandaserver/taskbuffer/workflow_processor.py", line 169, in core_exec\n    tmpLog)\n  File "/opt/panda/lib64/python3.6/site-packages/pandaserver/workflow/pcwl_utils.py", line 54, in parse_workflow_file\n    yaml_obj = yaml.main.round_trip_load(cwl_h, preserve_quotes=True)\n  File "/opt/panda/lib64/python3.6/site-packages/ruamel/yaml/main.py", line 1016, in round_trip_load\n    return load(stream, RoundTripLoader, version, preserve_quotes=preserve_quotes)\n  File "/opt/panda/lib64/python3.6/site-packages/ruamel/yaml/main.py", line 951, in load\n    return loader._constructor.get_single_data()\n  File "/opt/panda/lib64/python3.6/site-packages/ruamel/yaml/constructor.py", line 113, in get_single_data\n    return self.construct_document(node)\n  File "/opt/panda/lib64/python3.6/site-packages/ruamel/yaml/constructor.py", line 123, in construct_document\n    for _dummy in generator:\n  File "/opt/panda/lib64/python3.6/site-packages/ruamel/yaml/constructor.py", line 1563, in construct_yaml_map\n    self.construct_mapping(node, data, deep=True)\n  File "/opt/panda/lib64/python3.6/site-packages/ruamel/yaml/constructor.py", line 1469, in construct_mapping\n    value = self.construct_object(value_node, deep=deep)\n  File "/opt/panda/lib64/python3.6/site-packages/ruamel/yaml/constructor.py", line 146, in construct_object\n    data = self.construct_non_recursive_object(node)\n  File "/opt/panda/lib64/python3.6/site-packages/ruamel/yaml/constructor.py", line 188, in construct_non_recursive_object\n    for _dummy in generator:\n  File "/opt/panda/lib64/python3.6/site-packages/ruamel/yaml/constructor.py", line 1563, in construct_yaml_map\n    self.construct_mapping(node, data, deep=True)\n  File "/opt/panda/lib64/python3.6/site-packages/ruamel/yaml/constructor.py", line 1469, in construct_mapping\n    value = self.construct_object(value_node, deep=deep)\n  File "/opt/panda/lib64/python3.6/site-packages/ruamel/yaml/constructor.py", line 146, in construct_object\n    data = self.construct_non_recursive_object(node)\n  File "/opt/panda/lib64/python3.6/site-packages/ruamel/yaml/constructor.py", line 188, in construct_non_recursive_object\n    for _dummy in generator:\n  File "/opt/panda/lib64/python3.6/site-packages/ruamel/yaml/constructor.py", line 1563, in construct_yaml_map\n    self.construct_mapping(node, data, deep=True)\n  File "/opt/panda/lib64/python3.6/site-packages/ruamel/yaml/constructor.py", line 1470, in construct_mapping\n    if self.check_mapping_key(node, key_node, maptyp, key, value):\n  File "/opt/panda/lib64/python3.6/site-packages/ruamel/yaml/constructor.py", line 295, in check_mapping_key\n    raise DuplicateKeyError(*args)\nruamel.yaml.constructor.DuplicateKeyError: while constructing a mapping\n  in "/tmp/tmphqm69nyp/sP22a-hfcharm-workflow.cwl", line 13, column 5\nfound duplicate key "outputSource" with value "Pythia8Pass3Track/outDS" (original value: "Pythia8Calorimeter/outDS")\n  in "/tmp/tmphqm69nyp/sP22a-hfcharm-workflow.cwl", line 15, column 5\n\nTo suppress this check see:\n    http://yaml.readthedocs.io/en/latest/api.html#duplicate-keys\n\nDuplicate keys will become an error in future releases, and are errors\nby default when using the new API.\n\n\n\nERROR : workflow description is corrupted\n'
b''
submit:
2022-07-19 00:31:21.936128
b'INFO : uploading workflow sandbox\nINFO : gathering files under /direct/sphenix+u/jwebb2/work/2022/SHREK/sPHENIX-test-production/sP22a-hfcharm\nINFO : upload source files\nINFO : submit workflow user.jwebb2.sP22a-hfcharm-1896cb4c-06fa-11ed-9977-00163e101049\nINFO : request was accepted and will be processed in a few minutes\n'
b''